scripts/l10n/generate_po_files.py

```
@@ -93,6 +93,21 @@ def _get_license_header() -> str:
             + "\n#"
         )
 
+    @staticmethod
+    def _is_po_equal(pofile1: POFile, pofile2: POFile) -> bool:
+        """Compare two .po files considering comments."""
+        if len(pofile1) != len(pofile2):
+            return False
+
+        for entry1, entry2 in zip(pofile1, pofile2):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)